dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -271,6 +272,47 @@ protected Object loadField(String inode, String fieldContentlet) throws DotDataE
 	    return m.get(fieldContentlet);
 	}
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected Object loadJsonField(final String inode,
```

Comment:
Should the return here be a Optional<String>?

Reply:
It does not return Optional to be compliant with the original method called loadField but I guess you're right